core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -124,6 +125,7 @@ object GetOffsetShell {
     config.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList)
     config.setProperty(ConsumerConfig.CLIENT_ID_CONFIG, clientId)
     val consumer = new KafkaConsumer(config, new ByteArrayDeserializer, new ByteArrayDeserializer)
+    val client = Admin.create(config)
```

Comment:
I wonder if we could get rid of the `KafkaConsumer` now that the `Admin` client is used. It seems that it is used only to get the topics so that seems doable. What do you think?

Reply:
This is a good idea and it's easy, we only need to list topics and then describe topics. And I also find that we can optimize this by filter topics before describing topics, but this will need some extra work. What do you think?